src/lamp_py/mssql/mssql_utils.py

```diff
@@ -44,14 +44,15 @@ def get_local_engine(echo: bool = False) -> sa.future.engine.Engine:
             host=db_host,
             port=db_port,
             database=db_name,
-            query={
-                "driver": "ODBC Driver 18 for SQL Server",
-                "TrustServerCertificate": "yes",
-            },
         )
 
         engine = sa.create_engine(
             connection,
+            connect_args={
+                "driver": "ODBC Driver 18 for SQL Server",
+                "TrustServerCertificate": "yes",
+                "Encrypt": "no",
+            },
```

Comment:
I think this is the more correct location for these parameters. I added "Encrypt" = "no" because I was getting weird SSL Errors on repeated use of the MSSQLManager class.